### PR TITLE
[Importer] Fix importer for Parquet, ORC, and Avro formats

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- Fix external table creation with Parquet, Orc format
- Fix external and managed table creation with Avro format

## How was this patch tested?

- Manually
- Unit test

### CODE REVIEW

**Comment by agl29** [comment]
Please review the first commit ([1c3c83d570b5a1773d47f24406a520c994c8ed75]) to assess the main feature of the PR. The second commit contains lint fixes, which may distract from the main review focus.

File: desktop/libs/indexer/src/indexer/indexers/sql.py [hunk 1]
```diff
@@ -269,7 +258,7 @@ def create_table_from_a_file(self, source, destination, start_time=-1, file_enco
           extra_create_properties += "\nTBLPROPERTIES('transactional'='true', 'transactional_properties'='%s')" % \
               default_transactional_type
 
-      sql += '''\n\nCREATE TABLE `%(database)s`.`%(final_table_name)s`%(comment)s
+      sql += '''\n\nCREATE %(table_type)sTABLE `%(database)s`.`%(final_table_name)s`%(comment)s
```
Comment (quadoss) [inline desktop/libs/indexer/src/indexer/indexers/sql.py:261]: Is this correct where there is no space between %(table_type)sTABLE?

Comment (agl29) [reply desktop/libs/indexer/src/indexer/indexers/sql.py:261]: Yes, added space in 'EXTERNAL ' word. I added there as if table_type is empty then there will be 2 spaces and we don't want that.

**Review by Harshg999 — APPROVED** [review]
LGTM

**Review by JohanAhlen — APPROVED** [review]

**Pull request merged**